documentation/src/main/asciidoc/topics/proc_configuring_jgroups_asym_encrypt.adoc

```diff
@@ -0,0 +1,53 @@
+[id='configuring_jgroups_asym_encrypt-{context}']
+= Configuring Cluster Transport with Asymmetric Encryption
+Configure {brandname} clusters to generate and distribute secret keys that encrypt JGroups messages.
+
+.Procedure
+
+. Create a keystore with certificate chains that enables {brandname} to verify node identity.
+. Place the keystore on the classpath for each node in the cluster.
++
+For {brandname} Server, you put the keystore in the {server_home} directory.
```

Comment:
isn't `{server_home}/server/conf`?

Reply:
actually, use the same path as in https://infinispan.org/docs/stable/titles/server/server.html#configuring_infinispan_log_properties

```suggestion
For {brandname} Server, you put the keystore in the `{server_home}/${infinispan.server.root}/conf/` directory.
```

Reply:
@pruivo I think the keystore needs to be in $ISPN_HOME. if not you the server throws an exception.

Reply:
you're right, as usual :)